CHANGELOG.md

```diff
@@ -34,7 +34,7 @@
 ### Fixes and improvements
 
 * Fix empty translation returned by the REST translation server
-* Fix random split of protected sequences by BPE (#441)
+* Fix random split of protected sequences by BPE (#441) and vocabulary extraction (#444)
```

Comment:
This change was part of v0.9.3 and not v0.9.2. The changelog on master is correct.

Reply:
yes, not sure why I had this change - fixed